draft-ietf-quic-http.md

```diff
@@ -1004,11 +1004,15 @@ some higher layer of software that might have taken some action as a result. The
 client can treat requests rejected by the server as though they had never been
 sent at all, thereby allowing them to be retried later on a new connection.
 Servers MUST NOT use the HTTP_REQUEST_REJECTED error code for requests which
-were partially or fully processed.  When a client sends a STOP_SENDING with
-HTTP_REQUEST_CANCELLED, a server MAY indicate the error code
-HTTP_REQUEST_REJECTED in the corresponding RESET_STREAM if no processing was
-performed.  Clients MUST NOT reset streams with the HTTP_REQUEST_REJECTED error
-code except in response to a QUIC STOP_SENDING frame.
+were partially or fully processed.  When a server abandons a response after
+partial processing, it SHOULD abort its response stream with the error code
+HTTP_REQUEST_CANCELLED.
+
+When a client sends a STOP_SENDING with HTTP_REQUEST_CANCELLED, a server MAY
+indicate the error code HTTP_REQUEST_REJECTED in the corresponding RESET_STREAM
+if no processing was performed.  Clients MUST NOT reset streams with the
+HTTP_REQUEST_REJECTED error code except in response to a QUIC STOP_SENDING
+frame.
```

Comment:
```suggestion
frame that contains the same code.
```

The implication here is that a server won't use STOP_SENDING with REQUEST_REJECTED unless they have not processed anything.  That's not necessary obvious to me now, having had some time away from this subject.